doc/cli.markdown

```diff
@@ -672,6 +678,84 @@ Examples:
 
 device
 
+# Tags
+
+## tags
+
+Use this command to list all tags for
+a particular application, device or release.
+
+This command lists all application/device/release tags.
+
+Example:
+
+	$ balena tags --application MyApp
+	$ balena tags --device 7cf02a6
+	$ balena tags --release 1234
+
+### Options
+
+#### --application, -a, --app &#60;application&#62;
```

Comment:
What is the docs tool we're using? These encoded entities make output pretty much unreadable.